CONTRIBUTING.md

```diff
@@ -0,0 +1,297 @@
+# Contributing to OPM Simulators
+
+Thank you for your interest in contributing to the Open Porous Media (OPM) Simulators project! This guide will help you get started with contributing code, documentation, or bug reports.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Development Process](#development-process)
+- [Code Style and Formatting](#code-style-and-formatting)
+- [Automated Formatting Tools](#automated-formatting-tools)
+- [Testing](#testing)
+- [Submitting Changes](#submitting-changes)
+- [Reporting Issues](#reporting-issues)
+- [Getting Help](#getting-help)
+
+## Getting Started
+
+Before contributing, please:
+
+1. Read the [OPM Project Documentation](https://opm-project.org)
+2. Review the [OPM Manual](https://opm-project.org/?page_id=955)
```

Comment:
Wouldn't that take years? :sweat_smile: I am sure that I have only read part of it.

I don't think this is a prerequisite.